test/mero_test_util.erl

```diff
@@ -31,7 +31,7 @@
 -author('Miriam Pena <miriam.pena@adroll.com>').
 
 -export([start_server/5, stop_servers/1, wait_for_pool_state/5,
-         wait_for_min_connections_failed/4]).
+         wait_for_min_connections_failed/4, wait_for_cluster_mod/0]).
```

Comment:
You're exporting `wait_for_cluster_mod/0` but you're not using it anywhere other than this module.
Maybe we don't need to export it, actually.

Reply:
good catch, forgot to remove this 👍 tx